expressEngine/cms/customer/error.js

```diff
@@ -1,4 +1,6 @@
 export default {
   alreadyRegistered: 'You have already registered',
   customerNotFound: 'Customer not found',
+  notUpdated: 'Customer not updated',
+  emptyData: 'Nothing to update',
```

Comment:
@rahuljainss please place emptyData in common cms.

Reply:
done